bridge/autoload.php

```diff
@@ -1,12 +1,51 @@
 <?php
 
+// The autoloading mechanism is explained in `docs/autoloading.md`. If this file is changed, make sure that
+// documentation is up to date.
```

Comment:
Totally optional: What do you think about moving the docs out of `docs/autoloading.md` and into the relevant parts of this script as big comment blocks? I think having the docs and the code in the same file would really make it easier to understand and make it easier to keep the docs/comments up to date moving forward.